internal/provider/resource_tfe_policy_set.go

```diff
@@ -75,6 +75,18 @@ func resourceTFEPolicySet() *schema.Resource {
 				Default:  false,
 			},
 
+			"agent_enabled": {
+				Description: "Whether the policy set is executed in the TFC agent. True by default for OPA policies",
+				Type:        schema.TypeBool,
+				Optional:    true,
+			},
+
+			"policy_tool_version": {
+				Description: "The policy tool version to run the policy evaluation against",
+				Type:        schema.TypeString,
+				Optional:    true,
+			},
```

Comment:
Given these values can _also_ be set by the API make sure to set `Computed: True`.